.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: Feature Request
+description: Suggest an idea for for a new feature for Betaflight Configurator.
```

Comment:
Double "for" :)

Reply:
Maybe best: request a new feature idea for bet conf??

Reply:
The double 'for' was in the original file too XD

Fixed.

Reply:
yes, I see